telemetry/report/utility.py

```diff
@@ -0,0 +1,37 @@
+BOOT_PARTITION_BN_MAP = {
+    "zynq-adrv9364-z7020-bob-cmos" : "zynq-adrv9364-z7020-bob-vcmos",
+    "zynq-adrv9364-z7020-bob" : "zynq-adrv9364-z7020-bob-vlvds",
+    "zynqmp-zcu102-rev10-ad9081-204b-txmode9-rxmode4" : "zynqmp-zcu102-rev10-ad9081-v204b-txmode9-rxmode4",
+    "zynqmp-zcu102-rev10-ad9081-204c-txmode0-rxmode1" : "zynqmp-zcu102-rev10-ad9081-v204c-txmode0-rxmode1",
+    "zynqmp-zcu102-rev10-ad9081" : "zynqmp-zcu102-rev10-ad9081-vm4-l8",
+    "zynqmp-zcu102-rev10-ad9081-m8-l4" : "zynqmp-zcu102-rev10-ad9081-vm8-l4",
+    "zynq-zed-adv7511-adrv9002-rx2tx2" : "zynq-zed-adv7511-adrv9002-rx2tx2-vcmos",
+    "zynq-zed-adv7511-adrv9002" : "zynq-zed-adv7511-adrv9002-vcmos",
+    "zynqmp-zcu102-rev10-adrv9002-rx2tx2" : ["zynqmp-zcu102-rev10-adrv9002-rx2tx2-vcmos","zynqmp-zcu102-rev10-adrv9002-rx2tx2-vlvds"],
+    "zynqmp-zcu102-rev10-adrv9002" : ["zynqmp-zcu102-rev10-adrv9002-vcmos","zynqmp-zcu102-rev10-adrv9002-vlvds"],
+    "zynqmp-adrv9009-zu11eg-revb-adrv2crr-fmc-revb-jesd204-fsm":"zynqmp-adrv9009-zu11eg-revb-adrv2crr-fmc-revb",
```

Comment:
In boot partition, adrv9009-zu11eg is zynqmp-adrv9009-zu11eg-revb-adrv2crr-fmc-revb.